src/drivers/src/SteeringDriver.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * Created By: Gareth Ellis
+ * Created On: April 16, 2017
+ * Description: This node is responsible for passing twist messages received
+ *              over serial to the Arduino controlling the robot
+ */
+
+#include "SteeringDriver.h"
+
+/**
+ * Maps a given value from one range to another.
+ *
+ * If the given value is outside the input range, it will
+ * treated as a closest value in the input range (either the input min or max)
+ *
+ * @param input the value to be mapped
+ * @param input_min the minimum possible value of the input range
+ * @param input_max the maximum possible value of the input range
+ * @param output_min the minimum possible value of the output range
+ * @param output_max the maximum possible value of the output range
+ *
+ * @return the mapped value
+ **/
+void map(double& input, double input_min, double input_max,
+        double output_min, double output_max) {
+    if (input < input_min) input = input_min;
```

Comment:
could use "input = max(input, input_min);"

Reply:
im dumb keeping this as is